packages/core/src/index.ts

```diff
@@ -173,138 +174,126 @@ export const initialize = <T extends Record<string, any> = Record<string, any>>(
   appsConfig,
   api,
   options,
+  pluginStoreFeatureFlags = {},
+  pluginLoaderOptions = {},
+  pluginStoreOptions = {},
 }: {
   appsConfig: AppsConfig;
   api?: T;
   options?: Partial<ScalprumOptions>;
-}): void => {
+  pluginStoreFeatureFlags?: FeatureFlags;
+  pluginLoaderOptions?: Partial<PluginLoaderOptions>;
+  pluginStoreOptions?: Partial<PluginStoreOptions>;
```

Comment:
FYI, option types for `PluginStore` and `PluginLoader` are already using `Partial<T>`.

```ts
pluginLoaderOptions?: PluginLoaderOptions;
pluginStoreOptions?: PluginStoreOptions;
```

That's because we treat all `PluginStore` and `PluginLoader` options as optional by design.